d/directory-ldap-api/build_info.json

```diff
@@ -0,0 +1,20 @@
+{
+    "package_name": "directory-ldap-api",
+    "github_url": "https://github.com/apache/directory-ldap-api",
+    "version": "2.1.2",
+    "default_branch": "master",
+    "package_dir": "d/directory-ldap-api",
```

Comment:
@Pratikt2312 Can we move this script and folder to the format we intend to follow for all apache packages like apache-directory-ldap-api

Reply:
yes, sure.